src/main/java/io/debezium/connector/yugabytedb/connection/YBArray.java

```diff
@@ -0,0 +1,180 @@
+package io.debezium.connector.yugabytedb.connection;
```

Comment:
This file is for verifying the changes for the time being. IMO this is not even required as we already send the decoded values from the server, we simply need to parse those values properly which is what the current changes in `YugabyteDBValueConverter` is doing.